src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/config/ResourceServerConfiguration.kt

```diff
@@ -27,22 +21,11 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
         auth.antMatchers(
           "/webjars/**", "/favicon.ico", "/csrf",
           "/health/**", "/info",
-          "/v3/api-docs/**", "/api/swagger.json", "/swagger-ui/**",
-          "/swagger-resources", "/swagger-resources/configuration/ui",
-          "/swagger-resources/configuration/security"
+          "/v3/api-docs", "/swagger-ui/**",
+          "/swagger-resources", "/swagger-resources/configuration/ui", "/swagger-resources/configuration/security",
         ).permitAll()
           .anyRequest()
           .authenticated()
       }.oauth2ResourceServer().jwt().jwtAuthenticationConverter(AuthAwareTokenConverter())
   }
-
-  @Bean
```

Comment:
I assume you checked the API docs still work ok?

Reply:
Yeah - they still look okay